entity-framework/toc.yml

```diff
@@ -187,14 +191,18 @@
             href: core/querying/related-data/lazy.md
           - name: Related data and serialization
             href: core/querying/related-data/serialization.md
-      - name: Client vs. server evaluation
-        href: core/querying/client-eval.md
-      - name: Tracking vs. no-tracking
-        href: core/querying/tracking.md
       - name: Complex query operators
         href: core/querying/complex-query-operators.md
       - name: Raw SQL queries
         href: core/querying/raw-sql.md
+      - name: Database functions
```

Comment:
Maybe a sub-node for functions?

Reply:
Yes, I think these 3 topics can be collapsed unless you plan to write a lot in each of them

Reply:
> Yes, I think these 3 topics can be collapsed unless you plan to write a lot in each of them

A lot is coming soon.